debugtools/DDR_VM/data/superset-constants.dat

```diff
@@ -1542,7 +1542,6 @@ C|J9_EXCEPT_SEARCH_NO_HANDLER
 C|J9_EXCEPT_SEARCH_NO_UNWIND
 C|J9_EXTENDED_RUNTIME_ALLOW_APPLICATION_CONTENDED_FIELDS
 C|J9_EXTENDED_RUNTIME_ALLOW_CONTENDED_FIELDS
-C|J9_EXTENDED_RUNTIME_ALLOW_GET_CALLER_CLASS
```

Comment:
I don't believe this file should ever be modified by hand. @keithc-ca

Reply:
Ok I see. I'll wait for Keith, but won't force push a change yet since the builds are going. If all is well I'll revert this change and we can merge without wasting more machine resources re-running builds.

Reply:
It's safe to delete things from this file, in fact, I have some cleanup pending here that will remove all unused constants (they never needed to be here).